frontend/src/container/NewWidget/LeftContainer/QuerySection/index.tsx

```diff
@@ -114,11 +112,13 @@ function QuerySection({
 	);
 
 	const handleQueryCategoryChange = (qCategory: string): void => {
-		const currentQueryType = qCategory as EQueryType;
```

Comment:
Are we getting rid of this functionality?

Reply:
Yup, now the logic depends on queryType rather than creating an extra state variable.